devices.js

```diff
@@ -8060,6 +8060,31 @@ const devices = [
         description: 'FLAIR Viyu Smarte LED bulb RGB E27',
         extend: generic.light_onoff_brightness_colortemp_colorxy,
     },
+
+    // Alecto Smart Home
+    {
+        // this device needs its contact return value inverted from what 'fz.ias_contact_alarm_1' to work with domoticz
```

Comment:
If the contact is closed (so magnetic parts are against each other), what is the value of `contact`, `TRUE` or `FALSE`?

Reply:
This is the debug value of close:

```
debug 2020-02-29 15:44:43: Received Zigbee message from 'Door_Sensor', type 'commandStatusChangeNotification', cluster 'ssIasZone', data '{"zonestatus":0,"extendedstatus":0}' from endpoint 1 with groupID 0
```

and open then is:

```
debug 2020-02-29 15:44:43: Received Zigbee message from 'Door_Sensor', type 'commandStatusChangeNotification', cluster 'ssIasZone', data '{"zonestatus":1,"extendedstatus":0}' from endpoint 1 with groupID 0
```

Reply:
Then the added converter is not correct, you can just use `ias_contact_alarm_1`. When the contact is closed, contact should be TRUE (as contact is made), on open there is no contact, so contact should be FALSE. This is how it's currently implemented for all contact sensors.

Reply:
@Koenkk I would be inclined to agree but when i invert the value returned in the converter both the domoticz python zigbee2mqtt plugin and the homebridge siri integration project work as expected with this device.   Do you think the inversion of behavior should actually be done in the python plugin then ?

Reply:
It should indeed be done there, it's important to have this value consistent within zigbee2mqtt otherwise it becomes unmanageable

Reply:
I agree!  Will fix this with your input and push the changes this weekend.